src/geocat/f2py/rcm2points_wrapper.py

```diff
@@ -0,0 +1,212 @@
+import numpy as np
+import xarray as xr
+
+from dask.array.core import map_blocks
+from geocat.f2py.fortran import (drcm2points)
+from geocat.f2py.errors import (CoordinateError, ChunkError)
+from geocat.f2py.missing_values import (fort2py_msg, py2fort_msg)
+from .errors import (DimensionError)
+
+# Dask Wrappers _<funcname>()
+# These Wrapper are executed within dask processes, and should do anything that
+# can benefit from parallel excution.
+
+def _rcm2points(lat2d, lon2d, fi, lat1d, lon1d, msg_py, opt, shape):
+   
+    '''
+    Interpolates data on a curvilinear grid (i.e. RCM, WRF, NARR) to an unstructured grid.
+    
+    Args:
+        lat2d (:class:`numpy.ndarray`):
+    	    A two-dimensional array that specifies the latitudes locations
+    	    of fi. The latitude order must be south-to-north.
+            
+    	lon2d (:class:`numpy.ndarray`):
+    	    A two-dimensional array that specifies the longitude locations
+    	    of fi. The latitude order must be west-to-east.
+            
+    	fi (:class:`numpy.ndarray`):
+    	    A multi-dimensional array to be interpolated. The rightmost two
+    	    dimensions (latitude, longitude) are the dimensions to be interpolated.
+            
+    	lat1d (:class:`numpy.ndarray`):
+    	    A one-dimensional array that specifies the latitude coordinates of
+    	    the output locations.
+            
+    	lon1d (:class:`numpy.ndarray`):
+    	    A one-dimensional array that specifies the longitude coordinates of
+    	    the output locations.
+            
+    	opt (:obj:`numpy.number`):
+    	    opt=0 or 1 means use an inverse distance weight interpolation.
+    	    opt=2 means use a bilinear interpolation.
+            
+    	msg_py (:obj:`numpy.number`):
+    	    A numpy scalar value that represent a missing value in fi.
+    	    This argument allows a user to use a missing value scheme
+    	    other than NaN or masked arrays, similar to what NCL allows.
+    
+    Returns:
+    	:class:`numpy.ndarray`: The interpolated grid. A multi-dimensional array
+    	of the same size as fi except that the rightmost dimension sizes have been
+    	replaced by the number of coordinate pairs (lat1dPoints, lon1dPoints).
+    	Double if fi is double, otherwise float.
+    
+    Description:
+    	Interpolates data on a curvilinear grid, such as those used by the RCM (Regional Climate Model),
+    	WRF (Weather Research and Forecasting) and NARR (North American Regional Reanalysis)
+    	models/datasets to an unstructured grid. All of these have latitudes that are oriented south-to-north.
+    	A inverse distance squared algorithm is used to perform the interpolation.
+    	Missing values are allowed and no extrapolation is performed.
+    '''
```

Comment:
No need for docstring in inner wrapper, as the inner wrapper shouldn't be exposed to the end user.